.gitignore

```diff
@@ -73,3 +73,6 @@ tags
 
 # vim patches
 /vim-*.patch
+
+# ctest artifcats
+Testing
```

Comment:
should this go in `build/` (maybe `build/test/` or something)?

Reply:
I'm not sure I can control where this folder is created, but I can look it up to see if there's a way around it.

probably related: https://gitlab.kitware.com/cmake/cmake/-/issues/18156